docs/rules.md

```diff
@@ -1,129 +1,31 @@
 ---
 title: Rules
-redirectFrom:
-  - /docs/cli/built-in-rules/
-  - /docs/cli/resources/built-in-rules/
 ---
 
-# Rules
+# Rules to describe API standards
 
-All Redocly CLI rules are listed on this page.
+Redocly uses rules to describe all the different aspects of API behaviour that we check for during linting. Rules come in different levels of complexity, and can also be grouped for reuse. Here's an overview of what is available:
```

Comment:
Ah, yes, my English is rather mid-atlantic in places because I'm constantly switching dialects. Better tooling will pick this up for us instantly though, and save everyone time.